Dockerfile

```diff
@@ -7,11 +7,13 @@ RUN apt-get update \
         libsm6 \
         libxext6 \
         libxrender-dev \
+        libgl1-mesa-dev \
         python3.7-dev \
         python3-pip \
     && ln -s /usr/bin/python3.7 /usr/local/bin/python
 
-RUN python -m pip install --upgrade pip poetry setuptools
+# Pin setuptools to 49.x.x until this [issue](https://github.com/pypa/setuptools/issues/2350) is fixed.
+RUN python -m pip install --upgrade pip poetry setuptools==49.6.0
```

Comment:
Is this the key refactoring for this issue?

Reply:
What do you mean by "key refactoring"?

Reply:
If you only change this line, will that solve the issue?

Reply:
Yes. This one-line change will fix the issue. We pinned setuptools version before that was broken.